views/campaign-page.twig

```diff
@@ -89,7 +89,7 @@
                     </div>
                     <div id="datatable-container" class="col-sm-10">
                         <div class="XiboData">
-                            <table id="campaigns" class="table table-striped" data-content-type="campaign" data-content-id-name="campaignId" data-state-preference-name="campaignGrid">
+                            <table id="campaigns" class="table table-striped" data-content-type="campaign" data-content-id-name="campaignId" data-state-preference-name="campaignGrid" style="width: 100%;">
```

Comment:
Any reason we can't apply this to all `table` within `#datatable-container` via a stylesheet instead? I think that would fix resizing the window on other pages too?

Reply:
Datatables recommends adding the inline style property to have a flexible table: https://datatables.net/examples/basic_init/flexible_width.html

We can use something like this in our CSS:
`#datatable-container table.dataTable {
    width: 100% !important;
}`
But I'm not sure this will break anything in the datatable's flow :thinking: 
What do you think @dasgarner?